themes/default/content/registry/packages/pulumiservice/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Pulumi Cloud (Pulumi Service provider)
+title: Pulumi Service provider
```

Comment:
We should leave this as it was. We didn't change the name of the provider when we rebranded to Pulumi Cloud but we want to keep it clear that this is the Pulumi Cloud provider, it just has a different name.

Reply:
Ack, reverted to how it was.